release.Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM alpine:3.15.2
```

Comment:
how about using `debian:bullseye-slim` because musl sometimes can't hit it off with kubernetes or incompatible behaviors with glibc??

Reply:
OK, I'll use debian!

Reply:
I think ubuntu is more popular. But, ubuntu doesn't publish slim image, unfortunately...